tests/images.spec.ts

```diff
@@ -18,3 +24,47 @@ test("search for custom image and create an instance from it", async ({
 
   await deleteInstance(page, customInstance);
 });
+
+test("Export and Upload an image", async ({ page }) => {
+  // creating an image from an instance, because we can only export unified images
+  // and can't test the export with a standard image fetched from the image server
+  // we can remove this step once the export of split images is enabled by the backend.
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+  let imageAlias = await createImageFromInstance(page, instance);
+  const downloadPromise = page.waitForEvent("download");
+
+  await page.getByRole("link", { name: "Images", exact: true }).click();
+  await page.getByPlaceholder("Search").click();
+  await page.getByPlaceholder("Search").fill(imageAlias);
+
+  await expect(page.getByText(imageAlias)).toBeVisible();
+  let imageName = await getImageNameFromAlias(page, imageAlias);
+  await page.getByLabel("export image").click();
+  const download = await downloadPromise;
+  await page.waitForSelector(
+    `text=Image ${imageName} was successfully downloaded. Please check your downloads folder.`,
+  );
+  const IMAGE_FILE = "tests/fixtures/image.tar.gz";
+  await download.saveAs(IMAGE_FILE);
```

Comment:
```suggestion
```
I think we can get the location of the download directly, without manually saving it here (see other comment)